src/nodes.ts

```diff
@@ -96,23 +103,41 @@ export class CosmosDBAccountNode implements IMongoServer {
 			case "DocumentDB":
 				this.contextValue = "cosmosDBDocumentServer"
 				break;
+			case "Graph":
+				this.contextValue = "cosmosGraphDatabaseServer"
+				break;
 			default:
 				this.contextValue = "cosmosDBGenericResource";
 		}
+
 		this.collapsibleState = this.contextValue === "cosmosDBGenericResource" ? vscode.TreeItemCollapsibleState.None : vscode.TreeItemCollapsibleState.Collapsed;
 	}
 
 	get iconPath(): any {
-		if (this.defaultExperience === "MongoDB") {
-			return {
-				light: path.join(__filename, '..', '..', '..', 'resources', 'icons', 'light', 'DataServer.svg'),
-				dark: path.join(__filename, '..', '..', '..', 'resources', 'icons', 'dark', 'DataServer.svg')
-			};
-		} else {
-			return {
-				light: path.join(__filename, '..', '..', '..', 'resources', 'icons', 'theme-agnostic', 'Azure DocumentDB - DocDB account LARGE.svg'),
-				dark: path.join(__filename, '..', '..', '..', 'resources', 'icons', 'theme-agnostic', 'Azure DocumentDB - DocDB account LARGE.svg')
-			};
+		switch (this.defaultExperience) {
+			case "MongoDB":
+				return {
+					light: path.join(__filename, '..', '..', '..', 'resources', 'icons', 'light', 'DataServer.svg'),
+					dark: path.join(__filename, '..', '..', '..', 'resources', 'icons', 'dark', 'DataServer.svg')
+				};
+			case "Graph":
+				const graphPathAgnostic = path.join(__filename, '..', '..', '..', 'resources', 'icons', 'theme-agnostic', 'Azure DocumentDB - DocDB account LARGE.svg');
+				return {
+					light: graphPathAgnostic,
+					dark: graphPathAgnostic
+				};
+			case "DocumentDB":
+				const docDBPathAgnostic = path.join(__filename, '..', '..', '..', 'resources', 'icons', 'theme-agnostic', 'Azure DocumentDB - DocDB account LARGE.svg');
+				return {
+					light: docDBPathAgnostic,
+					dark: docDBPathAgnostic
+				};
+			default:
```

Comment:
Did you mean to add a file 'Azure DocumentDB - Generic account LARGE.svg'? If the icon doesn't exist yet, I recommend reverting this change until we have the icons.

Reply:
Now they're using the same icons as docdb (which PM is fine with).

Reply:
@StephenWeatherford what do you mean by now? This is what I see:
![screen shot 2017-11-07 at 11 53 37 am](https://user-images.githubusercontent.com/11282622/32514589-60ce6c96-c3b2-11e7-8a48-f166ca9074be.png)

Note: my table resource has no icon